src/fuzz/fuzz-varlink-idl.c

```diff
@@ -0,0 +1,32 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+#include <unistd.h>
+
+#include "errno-util.h"
+#include "fd-util.h"
+#include "fuzz.h"
+#include "io-util.h"
+#include "varlink-idl.h"
+#include "log.h"
+
+int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
+        _cleanup_(varlink_interface_freep) VarlinkInterface *vi = NULL;
+        _cleanup_free_ char *str = NULL, *dump = NULL;
+        int r;
+
+        if (!getenv("SYSTEMD_LOG_LEVEL"))
+                log_set_max_level(LOG_CRIT);
```

Comment:
why not do:

```c
log_set_max_level(LOG_CRIT);
log_parse_environment();
(void) log_open();
```
that way you first set the log level to LOG_CRIT, but then you allow the env var to override it

Reply:
I think it should set the log target to `LOG_TARGET_NULL` too. The fuzz targets shouldn't log anything on OSS-Fuzz (or in any other fuzzing environment) . That's probably something that should be fixed in all the fuzz targets though.